integration_tests/Cargo.toml

```diff
@@ -19,6 +19,7 @@ web_renderer = [
 ]
 
 [dependencies]
+compositor_api = { path = "../compositor_api" }
```

Comment:
add features to compositor_api and make sure it's passed to other `compositor_*` dependencies